app/views/devise/confirmations/new.html.erb

```diff
@@ -1,16 +1,16 @@
-<h2>Resend confirmation instructions</h2>
+<div class="users-confirmations-new container">
+  <h2>Resend Confirmation Instructions</h2>
 
-<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: {method: :post}) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+  <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: {method: :post}) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br>
-    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
-  </div>
+    <div class="field text mb-2">
+      <%= f.label :email %><br>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+    </div>
 
-  <div class="actions">
-    <%= f.submit "Resend confirmation instructions" %>
-  </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+    <div class="actions text-center">
+      <%= f.submit "Resend Confirmation Instructions", class: "btn btn-primary" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
Changed this template to look more like the other devise pages.

Reply:
Good catch!